projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -244,9 +253,23 @@ export class UpdateChanges {
                     fileContent = this.copyPropertyValueBetweenElementTags(fileContent, match, moveMatch);
                 }
 
+                if (change.valueTransform) {
+                    // We need to remove 'g' flag to see capture groups
+                    // This will FAIL if there are multiple bindings in A SINGLE TAG to the SAME property
+                    const regExpMatch = match.match(new RegExp(base));
+                    const boundProp = regExpMatch && regExpMatch[1] === ' [' ? InputPropertyType.EVAL : InputPropertyType.STRING;
```

Comment:
I'd take `endsWith('['` instead of hard equals in this case, just to proof this against future capture group changes.